CITATION.cff

```diff
@@ -0,0 +1,87 @@
+cff-version: 1.2.0
+title: >-
+  DAISY: A Data Information System for accountability
+  under the General Data Protection Regulation
+message: >-
+  If you use this software, please cite the paper
+  below.
+type: software
+authors:
+  - family-names: Becker
+    given-names: Regina
+    orcid: 'https://orcid.org/0000-0002-6711-8375'
+  - family-names: Alper
+    given-names: Pinar
+    orcid: 'https://orcid.org/0000-0002-2224-0780'
+  - family-names: Grouès
+    given-names: Valentin
+    orcid: 'https://orcid.org/0000-0001-6501-0806'
+  - family-names: Munoz
+    given-names: Sandrine
```

Comment:
Do you have the missing ORCIDs?

Reply:
I have contacted Sandrine - no reply so far. Otherwise, I will just proceed without her ORCID.